nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -296,6 +296,17 @@ in
         '';
       };
 
+      authorizedKeysInHomedir = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
I would personally go with not defaulting this to true and just adding a changelog entry. I assume that's the behaviour that most people expect and it's the same thing we did for `~/bin` some time ago.